corio.py

```diff
@@ -237,6 +238,11 @@ def monitor_processes(processes: dict) -> str or None:
             if tp_key == "health_check":
                 raise HealthCheckError(f"Process with PID {process.pid} stopped."
                                        f" Health Check collection error.")
+            if os.path.exists(log_path):
+                resp = run_local_cmd(f"grep 'topic {tp_key} completed successfully' {log_path} ")
```

Comment:
Check if there is any better way for checking completion of test.

Reply:
This will be logged in corio logger once all tests execution completed from respective workload, so this is one of the good check. Tested negative scenario as well and working fine.